### PR TITLE
Auctions

### PR DESCRIPTION
**Pull request by tomekczpog**
Added a !auction command
works like giveaway command but instead of choosing random person it starts a bidding war
if someone bids when the auction almost ended the time gets extended
also you can't spend your bobux reserved for auctions if u have some in auction

### CODE REVIEW

**Comment by tomekczpog** [comment]
deez nuts

File: forumsweats/commands/auction.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+	await update_auction_message(message)
+	await end_auction(data)
+
+
+def create_auction_embed(data: dict, bidder=None):
+	item = data['item']
+	creator_id = data['creator_id']
+	end = data['end']
+	highest_bidder = bidder or data.get("highest_bidder")
```
Comment (mat-1) [inline forumsweats/commands/auction.py:106]: Use single quotes please

File: forumsweats/commands/auction.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+			timeout=60
+		)
+
+		if m.content.lower() == 'cancel':
+			return await m.add_reaction('👍')
+		
+		user_response = await check(m.content)
+
+		if user_response is None:
+			await channel.send(invalid_message + ' (Type "cancel" to cancel the giveaway creation)')
```
Comment (mat-1) [inline forumsweats/commands/auction.py:218]: You forgot to change this to auction instead of giveaway

Comment (tomekczpog) [reply forumsweats/commands/auction.py]: Both fixed

**Pull request merged**